src/store.js

```diff
@@ -24,7 +23,7 @@ const state = {
   },
   platform: 'pc',
   theme: 'night',
-  grid: grid,
+  grid: {},
```

Comment:
do we still need this variable?

Reply:
i was gonna store the new grid in it for ease.

Reply:
like positions, and whatnot

Reply:
I can delete for now since we won't use it for a while